app/src/main/java/com/codesoom/assignment/core/application/MemberService.java

```diff
@@ -0,0 +1,68 @@
+package com.codesoom.assignment.core.application;
+
+import com.codesoom.assignment.core.domain.Member;
+import com.codesoom.assignment.core.domain.MemberRepository;
+import com.codesoom.assignment.web.dto.MemberData;
+import com.codesoom.assignment.web.exception.MemberNotFoundException;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+@Transactional
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    public MemberService(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    /**
+     * 신규 회원를 등록합니다.
+     * @param  memberData - 신규 등록할 회원 정보(이름, 비밀번호, 이메일 등).
+     * @return member - 고유 ID를 발급받은 신규 회원.
+     */
```

Comment:
스탠다드 라이브러리의 JavaDoc 주석들이 `Returns`로 가득했던 것을 떠올려 보세요. 리턴값에 대한 설명이 가장 중요합니다. 처음 보는 매우 복잡한 코드가 있다고 가정해 봅시다. 그때 "무엇을 넣으면 무엇이 나온다"는 주석이 위에 있다면 엄청 반가울 거에요. 그 주석이 사실이라면 그만큼 도움되는 주석도 없을 겁니다.

```suggestion
    /**
     * 신규 회원을 등록하고, 등록된 회원을 리턴합니다.
     * @param  memberData 신규 등록할 회원 정보
     * @return member 저장된 신규 회원
     */
```

`@param`에 달린 설명은 필요 이상으로 자세했어요. 이름, 비밀번호, 이메일은 지금은 `MemberData`에 소속된 필수정보지만 만약 보안 이슈가 생겨서 비밀번호를 다른 클래스로 분리하게 됐다고 생각해 봅시다. 그렇다면 이 클래스의 이 메소드까지 일부러 찾아와서 저 `비밀번호`라는 주석을 삭제해 주는 귀찮은 일을 해야겠죠. 주석과 비밀번호 필드 사이에 커플링이 생겨 버린 것입니다. 주석에도 객체지향 원칙이 적용된다는 점을 꼭 기억해 주세요.